src/sentry/tasks/groupowner.py

```diff
@@ -50,7 +50,7 @@ def _process_suspect_commits(
                 "sentry.tasks.process_suspect_commits.get_serialized_event_file_committers"
             ):
                 committers = get_event_file_committers(
-                    project, group_id, event_frames, event_platform
+                    project, group_id, event_frames, event_platform, sdk_name
```

Comment:
You need to pass this as a kwarg (might be safer to pass as a kwarg in other places that you pass it too, but not causing bugs currently)

Reply:
Why does this need to be passed as a kwarg? The `sdk_name` is optional. 

I'm fine with adding it, just don't fully understand why its needed.

Reply:
It's what's causing your test failures, since currently you're passing it to the `frame_limit` arg.

Reply:
Ahh, doh...